### ISSUE TITLE
sc-editor - grows infinitely when given 100% height inside a host without set height 

### ISSUE THREAD

**Issue by agolvet** [ircam-ismm/sc-components#15]
```
<div>
   <sc-editor style="height: 100%></sc-editor>
</div>
```

may only happen in some cases

**Comment by b-ma** [comment]
just had the same with `sc-signal` container was `flex` with no height given

**Comment by b-ma** [comment]
Seems to be when the container is `display: flex` and `width` or `height` set to 100%

See if it is fixed with #49

**Comment by b-ma** [comment]
Fixed w/ https://github.com/ircam-ismm/sc-components/issues/49
